quetzal/prisma/schema.prisma

```diff
@@ -8,15 +8,66 @@ datasource db {
 }
 
 model users {
-  id             Int       @id @default(autoincrement())
-  pk             Int?      @unique @db.Integer
-  slack_user_id  String    @unique @db.VarChar(255)
-  tilde_username String    @unique @db.VarChar(255)
-  name           String?   @db.VarChar(255)
-  email          String?   @db.VarChar(255)
+  id             Int        @id @default(autoincrement())
+  pk             Int?       @unique @db.Integer
+  slack_user_id  String     @unique @db.VarChar(255)
+  tilde_username String     @unique @db.VarChar(255)
+  name           String?    @db.VarChar(255)
+  email          String?    @db.VarChar(255)
   ssh_public_key String
   description    String
-  is_approved    Boolean?  @default(false)
+  is_approved    Boolean?   @default(false)
   message_id     String?
-  created_at     DateTime? @default(dbgenerated("'1970-01-01 00:00:00+00'::timestamp with time zone")) @db.Timestamptz(6)
+  created_at     DateTime?  @default(dbgenerated("'1970-01-01 00:00:00+00'::timestamp with time zone")) @db.Timestamptz(6)
+  admin          Boolean    @default(false)
+  nominees       nominees[]
+  ballots        ballots[]
```

Comment:
How will this handle multiple elections?

Reply:
The nominees and ballots themselves link to an election; this relation isn't explictly queried